src/pages/Loans/LoansOverview/components/CollateralModal/CollateralModal.tsx

```diff
@@ -59,49 +66,90 @@ const CollateralModal = ({ asset, position, onClose, ...props }: CollateralModal
   const { getLTV } = useGetLTV();
   const prices = useGetPrices();
 
+  const overlappingModalRef = useRef<HTMLDivElement>(null);
+
   const transaction = useTransaction({
     onSigning: onClose,
     onSuccess: refetch
   });
 
-  if (!asset || !position) {
-    return null;
-  }
-
   const { isCollateral: isCollateralActive, amount: lendPositionAmount } = position;
 
   const loanAction = isCollateralActive ? 'withdraw' : 'lend';
   const currentLTV = getLTV({ type: loanAction, amount: lendPositionAmount });
   const variant = getModalVariant(isCollateralActive, currentLTV?.status);
 
-  const content = getContentMap(t, variant, asset);
-
-  const handleClickBtn = () => {
-    if (variant === 'disable-error') {
-      return onClose?.();
-    }
-
+  const handleSubmit = () => {
     if (variant === 'enable') {
       return transaction.execute(Transaction.LOANS_ENABLE_COLLATERAL, asset.currency);
     } else {
       return transaction.execute(Transaction.LOANS_DISABLE_COLLATERAL, asset.currency);
     }
   };
 
+  const form = useForm<ToggleCollateralLoansFormData>({
+    initialValues: {
+      [LOANS_TOGGLE_COLLATERAL_FEE_TOKEN_FIELD]: ''
```

Comment:
Shouldn't this be
```suggestion
      [LOANS_TOGGLE_COLLATERAL_FEE_TOKEN_FIELD]: transaction.fee.defaultCurrency.ticker
```

Reply:
No, this is the same problem of the previous PR. The form does not have any input, so it does not require you to fill something to submit the form. This will be solved later on.

Reply:
Ok, then let's note it somewhere, what about adding TODO comment? :)